### DISCUSSION TITLE
Why avif webp images are shranked when converted from tiff in bitmap mode?

### DISCUSSION THREAD

**Discussion by joesong168** [libvips/libvips#3682] [Q&A]
## Questions, enhancements, tips, etc.

Why avif webp images are shranked when converted from tiff in bitmap mode
## Bug report

**Describe the bug**

vips copy test2.tif output.avif
[test2.tif.zip](https://github.com/libvips/libvips/files/12702907/test2.tif.zip)

**Comment by jcupitt** [comment]
Hi again,

```
$ file test2.tif
test2.tif: PDF document, version 1.1, 1 pages
```

It's not a TIFF, your file is a PDF. They get loaded at 72 DPI by default, use some other DPI setting if you need a larger file, eg.:

```
$ vips copy test2.tif[dpi=600] x.avif
$ vipsheader x.avif 
x.avif: 6426x8960 uchar, 4 bands, srgb, heifload
```
